### ISSUE TITLE
[Modpack Problem]: Some mod resource blocks do not spawn as ores on the "Balanced" planet.

### ISSUE THREAD

**Issue by AutumnalModding** [FTBTeam/FTB-University#124]
* What version of University: 1.1.0
* Describe the issue: Thaumcraft's "Brass Block", "Thaumium Block", and "Void Metal Block" do not spawn in the "Balanced" dimension. Additionally, Enderium Blocks, Signalum Blocks,  Lumium Blocks (from Thermal Foundation) and EnderIO/Endergy resource blocks do not spawn in the "Balanced" dimension.

**Comment by NuclearDogs-1** [comment]
Why would enderium, signalum or lumium need to spawn in Balanced.

**Comment by AutumnalModding** [comment]
...Grinding balls? :p

**Comment by NuclearDogs-1** [comment]
they aren't that bad to make

**Comment by NuclearDogs-1** [comment]
Have you progressed far enough in thaumcraft to be able to get access to brass, thaumium and void metal?

**Comment by AutumnalModding** [comment]
I have progressed enough to unlock "Delving into the Eldritch" research, however I have not completed it yet.

**Comment by aaronhowser1** [comment]
```
				<ore block="thaumcraft:metal_brass" minHeight="20" maxHeight="80" clumpSize="5" chancePerChunk="100" />
				<ore block="thaumcraft:metal_thaumium" minHeight="20" maxHeight="80" clumpSize="5" chancePerChunk="50" />
				<ore block="thaumcraft:metal_void" minHeight="20" maxHeight="80" clumpSize="5" chancePerChunk="25" />
				<ore block="enderio:block_alloy" meta="0" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="1" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="2" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="3" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="4" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="5" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="6" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="7" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="8" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy" meta="9" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_infinity" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy_endergy" meta="0" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy_endergy" meta="1" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy_endergy" meta="2" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="50" />
				<ore block="enderio:block_alloy_endergy" meta="3" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="20" />
				<ore block="enderio:block_alloy_endergy" meta="5" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="enderio:block_alloy_endergy" meta="6" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="environmentaltech:litherite" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
				<ore block="environmentaltech:pladium" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="70" />
				<ore block="environmentaltech:ionite" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="50" />
				<ore block="environmentaltech:aethium" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="40" />
				<ore block="environmentaltech:kyronite" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="80" />
				<ore block="environmentaltech:erodium" minHeight="0" maxHeight="10" clumpSize="5" chancePerChunk="100" />
```

**Comment by AutumnalModding** [comment]
Closing as fixed. Thanks!

**Comment by aaronhowser1** [comment]
It isn't fixed yet, 1.2 isn't out.
